newlib/libc/include/sys/_types.h

```diff
@@ -144,7 +144,13 @@ typedef __uint32_t __mode_t;
 __extension__ typedef long long _off64_t;
 #endif
 
-typedef _off_t __off_t;
+#include <stdint.h>
```

Comment:
We can't include stdint.h here -- that would expose all of those symbols to anything using _types.h.